packages/react-arborist/src/components/default-container.tsx

```diff
@@ -73,44 +73,43 @@ export function DefaultContainer() {
           focusPrevElement(e.currentTarget);
           return;
         }
-        if (e.key === "ArrowDown" && !e.shiftKey && !e.metaKey) {
+        if (e.key === "ArrowDown") {
```

Comment:
This is looking great! The reason the metaKey check was there was because of another case further down in the file: https://github.com/brimdata/react-arborist/blob/12c54bfe4027ce78b89577b2154be880b8f01041/packages/react-arborist/src/components/default-container.tsx#L184-L189

If you press "meta+DownArrow" it "activates" that node. (This is something VSCode and Mac Finder do).

Why don't you move the body of that if statement here where all the rest of the "ArrowDown" cases live. Then I think we'd be good to go. 

Thanks! I appreciate it!

Reply:
Oops again, thanks for catching that!